java/code/src/com/redhat/rhn/frontend/action/errata/test/NotifySetupActionTest.java

```diff
@@ -52,22 +51,10 @@ public void testExecute() throws Exception {
         User user = requestContext.getCurrentUser();
         Errata published = ErrataFactoryTest
                 .createTestPublishedErrata(user.getOrg().getId());
-        Errata unpublished = ErrataFactoryTest
-                .createTestUnpublishedErrata(user.getOrg().getId());
-
-        //test bad parameter exception
-        request.setupAddParameter("eid", unpublished.getId().toString());
-        try {
-            action.execute(mapping, form, request, response);
-            fail();
-        }
-        catch (BadParameterException e) {
-            //Success!!!
-        }
 
         //test default case
         request.setupAddParameter("eid", published.getId().toString());
-        request.setupAddParameter("eid", unpublished.getId().toString());
+        request.setupAddParameter("eid", published.getId().toString());
```

Comment:
seems you have this line now 2 times

Reply:
One really needs two in this case, as `request` is a mock object and the number of times the parameter is read counts!

Reply:
ok